_layouts/bib.liquid

```diff
@@ -261,12 +272,14 @@
               data-hide-less-than="15"
               data-badge-type="2"
               data-badge-popover="right"
-              {% if entry.eprint %}
+              {% if entry.altmetric != blank and entry.altmetric != 'true' %}
+                data-altmetric-id="{{ entry.altmetric }}"
+              {% elsif entry.arxiv %}
```

Comment:
Since we are changing to also check for blank for that field, shouldn't we change it for all of them? What do you think?

Reply:
I think you are right. The check for blank string is not necessary. I previously thought that `entry.altmetic` will be set to an empty string whenever it is provided in the BibTex. A new commit fixed this.